crates/holochain_sqlite/src/sql/cell/schema/3-up.sql

```diff
@@ -0,0 +1,4 @@
+ALTER TABLE
+  ChainLock RENAME COLUMN author TO subject;
+ALTER TABLE
+  ChainLock RENAME COLUMN lock TO author;
```

Comment:
So, to my understanding. `0.sql` is used, and `#-up.sql` are used. We had been creating `#.sql` files, but they were just for reference?? We're not bothering creating a `3.sql` here. Should we just delete the `#.sql` files > 0 and be done with it? Or am I not understanding the flow correctly?

Reply:
You are understanding correctly. It's a way of seeing what SQL should create the latest state of the database, assuming that the `#.sql` was written correctly. It's safest to connect to the database and see what the current schema really is after the migrations have run. That's the way I've always worked with migration tools.

I would prefer to delete those reference files, but I won't unless @maackle is okay with that?

Reply:
If the majority of us want to remove the *.sql reference files, I'll accept that, but I personally prefer to have them. 

Even though we're not programmatically checking the correctness of our reference schemas, having them at all shows that we expect to know what migration we actually did run, and if in the course of normal work we find that the actual schema deviates from the reference schema, that is useful information, because it doesn't necessary mean that the actual schema is correct, but perhaps the migration was wrong. We lose that information if we only check the (potentially wrong) schema.

Reply:
Also, merely querying the DB for the current schema means we don't have sensible comments for the fields